HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -10,6 +10,7 @@ public enum Hug2Status
         NotParticipating,
         NoLongerParticipating,
```

Comment:
I wonder about marking these two as `Obsolete` now, so usages will get flagged?

Reply:
could do, though since there's a lot of existing usages in routing worry about the number of false positives

Reply:
We can suppress the warnings we expect, with a view to removing them permanently at some point (soon). It means we'd catch any LAs that were accidentally marked with one of these obsolete statuses, which I think would be a sufficiently bad thing that we want to have safeguards against it.

Reply:
Alternatively, we could get agreement from DESNZ that they're not going to use either of these statuses ever again (which feels likely) and completely remove them now.

Reply:
to confirm, the view I thought we agreed at standup is that we weren't going to do much work at this time to removing these deprecated statuses, since HUG2 doesn't have a long term future. is this something we'd now like to do?

Reply:
HUG2 doesn't but we're going to be using the same code for WHLG. So let's not remove them until we've agreed all the new statuses and associated copy. We should mark as obsolete given we're not removing, to make sure we don't have any LAs slip through.

Reply:
have marked as obsolete